src/main/java/net/kodehawa/mantarobot/commands/currency/pets/HousePetType.java

```diff
@@ -31,6 +31,7 @@
             List.of(HousePetAbility.CHEER, HousePetAbility.FISH, HousePetAbility.CATCH, HousePetAbility.CHOP),
             3000000, 300, true
     ),
+    ROCK(EmoteReference.ROCK, "Rock", List.of(HousePetAbility.CHEER), 1, 1, true),
```

Comment:
might wanna make the value 1000 or smth (100, 0)

Reply:
Done.